Templates/SaaSFileTrigger-Batch/run.bat

```diff
@@ -0,0 +1,5 @@
+@echo off
+
+type %input% >> %output%
+
+echo bat file processed a file!
```

Comment:
nit: use the same format the other Batch samples use "Windows Batch script processed ..."